lib/build.api.js

```diff
@@ -32,8 +32,7 @@ class Build {
       symlink: true,
       delPatterns: ['*', '.*'],
       excludeLinks: [],
-      addLinks: [],
-      makeFile: this.getAbsolutePath('drupal.make.yml')
+      addLinks: []
```

Comment:
Out of curiosity was it intentional to use `makeFile` in the build api and `make` in the project api?